pkg/security/probe/constantfetch/fallback.go

```diff
@@ -563,3 +573,79 @@ func getNFConnCTNetOffset(kv *kernel.Version) uint64 {
 
 	return offset
 }
+
+func getSockCommonSKCFamilyOffset(kv *kernel.Version) uint64 {
+	return 16
+}
+
+func getFlowi4SAddrOffset(kv *kernel.Version) uint64 {
+	offset := uint64(40)
+
+	switch {
+	case kv.IsRH7Kernel():
+		offset = 40
+	case kv.IsRH8Kernel():
+		offset = 56
+
+	case kv.IsInRangeCloseOpen(kernel.Kernel5_0, kernel.Kernel5_1):
+		offset = 32
+	case kv.Code >= kernel.Kernel5_1:
+		offset = 40
+	}
+
+	return offset
+}
+
+func getFlowi4ULIOffset(kv *kernel.Version) uint64 {
+	offset := uint64(48)
+
+	switch {
+	case kv.IsRH7Kernel():
+		offset = 48
+	case kv.IsRH8Kernel():
+		offset = 64
+
+	case kv.IsInRangeCloseOpen(kernel.Kernel5_0, kernel.Kernel5_1):
+		offset = 40
+	case kv.Code >= kernel.Kernel5_1:
+		offset = 48
+	}
+
+	return offset
+}
+
+func getFlowi6SAddrOffset(kv *kernel.Version) uint64 {
+	offset := uint64(56)
+
+	switch {
+	case kv.IsRH7Kernel():
+		offset = 56
+	case kv.IsRH8Kernel():
+		offset = 72
+
+	case kv.IsInRangeCloseOpen(kernel.Kernel5_0, kernel.Kernel5_1):
+		offset = 48
+	case kv.Code >= kernel.Kernel5_1:
+		offset = 56
+	}
+
+	return offset
+}
+
+func getFlowi6ULIOffset(kv *kernel.Version) uint64 {
+	offset := uint64(76)
```

Comment:
isn't it getFlowi6SAddrOffset + 20 ?

Reply:
Updating ... I hesitated because I didn't know if we'd get different values before running the CI :)